drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -225,51 +227,96 @@ Error AudioDriverPulseAudio::init_device() {
 	return OK;
 }
 
+#define LIBPULSE_DLSYM(name)                                                 \
+	libpulse_##name = (name##_t)dlsym(libpulse, #name);                      \
+	if (!libpulse_##name) {                                                  \
+		fprintf(stderr, "could not find symbol %s: %s\n", #name, dlerror()); \
+		return FAILED;                                                       \
+	}
+
 Error AudioDriverPulseAudio::init() {
 
+	libpulse = dlopen("libpulse.so", RTLD_LAZY);
+	if (!libpulse) {
+		fprintf(stderr, "could not load libXrandr.so.2, Error: %s\n", dlerror());
```

Comment:
Wrong error message :) You could likely use `ERR_PRINTS` or ` ERR_FAIL_COND_V` too.